docs/02-architecture/06-client-side-architecture.md

```diff
@@ -153,9 +153,7 @@ Gets the [Ophan browserId](https://github.com/guardian/frontend/blob/main/common
 
 ### Analytics
 
-The analytics for Dotcom are defined in [analytics/base.scala.html](https://github.com/guardian/frontend/blob/main/common/app/views/fragments/analytics/base.scala.html). It contains [Google Analytics](https://github.com/guardian/frontend/blob/main/common/app/views/fragments/analytics/google.scala.html), [Omniture](https://github.com/guardian/frontend/blob/main/common/app/views/fragments/analytics/omniture.scala.html) and [Comscore](https://github.com/guardian/frontend/blob/main/common/app/views/fragments/analytics/comscore.scala.html).
```

Comment:
I tried to check if we have any other analytics for omniture or comscore but none of them in `base.scala.html` so I removed them but please correct me if I missed something!